lib/loans/subgraph/subgraphLoans.ts

```diff
@@ -176,10 +187,18 @@ export async function getMostRecentTermsForLoan(
   return lendEvents.sort((a, b) => b.blockNumber - a.blockNumber)[1];
 }
 
-export async function getCreatedLoansSince(timestamp: number) {
+export async function getCreatedLoansSince(
+  timestamp: number,
+  nftBackedLoansSubgraph?: string,
```

Comment:
we should be making this param required right? i can do in a follow up PR

Reply:
Yep, just optional here since some of the changes are dependent on your other PR